Makefile

```diff
@@ -256,12 +149,11 @@ OCTP_DEPS = \
 src/subtitles-octopus-worker.bc: $(OCTP_DEPS) all-src
 .PHONY: all-src
 all-src:
-	PKG_CONFIG_PATH=$(DIST_DIR)/lib/pkgconfig \
 	$(MAKE) -C src all
 
 # Dist Files
 EMCC_COMMON_ARGS = \
-	$(GLOBAL_LDFLAGS) \
+	$(LDFLAGS) \
```

Comment:
There's no C or C++ code being compiled here. Only prebuilt object files and pre- & post-js being linked together.

Reply:
Ah, I guess I should re-phrase this then, as I'm not sure about the structure of how this compiles anymore.
Is SubtitlesOctopus.cpp and libass.cpp compiled with `-O3`? because last time I tried it wasn't, and this potentially leaves some performance on the table

Reply:
It is by default. See `src/Makefile`.